app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,43 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+
+    if @post.save
+      redirect_to @post
+    else
```

Comment:
We will usually just roll with the pattern of `Post.create!(...)`, but this works

Reply:
Oh, I forgot that was a thing. Will change, thanks!